### PR TITLE
Add tests for `CertificatesOps` `add_certificates_batch` `revoked`

### LINKED ISSUE #5470: Tests of `add_certificates_batch` for `RevokedUserCertificate`

### PR DESCRIPTION
**Pull request by TimeEngineer**
closes #5470

### CODE REVIEW

File: libparsec/crates/client/tests/unit/certificates_ops/add_revoked_certificate.rs [hunk 1]
```diff
@@ -0,0 +1,183 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
```
Comment (touilleMan) [inline libparsec/crates/client/tests/unit/certificates_ops/add_revoked_certificate.rs:1]: better to name this file after the actual certificate name (so `add_user_revoked_certificate.rs`)

File: libparsec/crates/client/src/certificates_ops/add.rs [hunk 2]
```diff
@@ -102,7 +102,7 @@ pub enum AddCertificateError {
     Internal(#[from] anyhow::Error),
 }
 
-#[derive(Debug)]
+#[derive(Debug, PartialEq, Eq)]
```
Comment (touilleMan) [inline libparsec/crates/client/src/certificates_ops/add.rs:105]: please staaaap :'(

**Review by touilleMan — CHANGES_REQUESTED** [review]

File: libparsec/crates/client/tests/unit/certificates_ops/add_revoked_certificate.rs (hunk 1 above)
Comment (TimeEngineer) [reply libparsec/crates/client/tests/unit/certificates_ops/add_revoked_certificate.rs:1]: Done

File: libparsec/crates/client/src/certificates_ops/add.rs (hunk 2 above)
Comment (TimeEngineer) [reply libparsec/crates/client/src/certificates_ops/add.rs]: Done

Comment (TimeEngineer) [reply libparsec/crates/client/src/certificates_ops/add.rs]: That's why I don't like to open multiple PR, when someone review something, I have to change everywhere

**Review by touilleMan — APPROVED** [review]

**Pull request merged**